src/main/java/control/ControllerManager.java

```diff
@@ -0,0 +1,29 @@
+package control;
+
+import gui.RootPane;
+import lombok.Getter;
+
+/**
+ * Class wrapper for model management controllers.
+ * @author alex
+ */
+public class ControllerManager {
+
+    private RootPane rootPane;
+
+    @Getter
+    private TimelineController timelineControl;
+
+    /**
+     * Constructor.
+     * @param rootPane Root Window
+     */
+    public ControllerManager(RootPane rootPane) {
+        this.rootPane = rootPane;
+        initializeControllers();
+    }
+
+    private void initializeControllers() {
+        timelineControl = new TimelineController(rootPane.getRootCenterArea().getGrid(), rootPane);
```

Comment:
Why do you pass both the rootPane and the grid to the constructor. The grid can be fetched from the rootPane inside the constructor as well